repotracker/repotracker.go

```diff
@@ -1110,6 +1112,19 @@ func createVersionItems(ctx context.Context, v *model.Version, metadata model.Ve
 			}
 			return errors.Wrapf(err, "error inserting version '%s'", v.Id)
 		}
+		// todo uncomment
+		//_, err = db.Collection(model.ProjectConfigsCollection).InsertOne(sessCtx, projectInfo.IntermediateConfig)
```

Comment:
We should only insert this if IntermediateConfig isn't nil, and we should only set IntermediateConfig if anything exists for it (may want an IsEmpty function or something that checks all of the things besides ID? Not sure)

Reply:
Implemented isEmpty for the project configs to prevent this.